python/feathub/feature_views/feature_view.py

```diff
@@ -52,10 +52,11 @@ def __init__(
                        string, it should refer to the name of a table descriptor in the
                        registry.
         :param features: A list of features to be joined onto this feature view.
-                         If a feature is a string, it should be either in the format
-                         {table_name}.{feature_name}, which refers to a feature in the
-                         table with the given name, or in the format {feature_name},
-                         which refers to a feature in the source table.
+                         If a feature is a string, it should be it should be in either
+                         of the following formats:
+                         1. {feature_name}
```

Comment:
Can you add the comment to explain that `{feature_name}` refers to a feature in the source table?

Same for `OnDemandFeatureView`.